step.sh

```diff
@@ -35,5 +35,5 @@ if [ "$fdu_logging" = "yes" ]; then
   echo ""
   echo
 fi
-
+chmod +x $fdu_google_services_location
```

Comment:
I have never seen this issue occur, some more insight into where it is happening and the steps to reproduce will be required.

I would suggest you include the following information:

- Is the occurring locally, on Bitrise, or your own enterprise installation?
- What version of FirebaseCrashlytics are you using?
- What versions of Xcode and macOS (eg the build stack on Bitrise) are you using?

Reply:
hey thanks, @andrewmarmion i'll check it out